platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -92,7 +92,7 @@ int _odp_ipsec_status_send(odp_queue_t queue,
 #define ODP_CONFIG_IPSEC_SAS	8
 
 struct ipsec_sa_s {
-	odp_atomic_u32_t state ODP_ALIGNED_CACHE;
+	odp_atomic_u32_t ODP_ALIGNED_CACHE state;
```

Comment:
The new attribute placement rule was for structs, union, etc, but how about integers ?

Reply:
The real bug was only present on typedef since the attribute was ignored if  placed after the terminating }.
typedef unsigned long long __u64 __attribute__((aligned(16))); 
should be equal to 
typedef unsigned long long  __attribute__((aligned(16))) __u64;